onnx/defs/tensor/defs.cc

```diff
@@ -8,6 +8,7 @@
 using namespace ONNX_NAMESPACE;
 
 ONNX_OPERATOR_SCHEMA(Cast)
+    .SinceVersion(6)
```

Comment:
6 or 7?

Reply:
We introduced this change in opset 6, I think it's better to keep the truth in our code.

Reply:
Any particular reason to use 7? If there is, we can use 7.

Reply:
This should be 6, which was the time we introduced this op.